pkg/genericapiserver/options/server_run_options.go

```diff
@@ -73,6 +73,7 @@ type ServerRunOptions struct {
 	EnableSwaggerUI           bool
 	EnableWatchCache          bool
 	EtcdServersOverrides      []string
+	ExpectedRAMCapacityMB     int
```

Comment:
Hrm, this is also not great. Maybe "TargetRAMUsageMB"?

"Expected" is a weird word here, the system should definitely know how much ram there is before it runs this.